profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java

```diff
@@ -0,0 +1,413 @@
+//		return managementRate;
+//	}
+//
+//	public void setManagementRate(ManagementRate managementRate) {
+//		this.managementRate = managementRate;
+//	}
+//
+//	public AutoOrManualValue getRxCellSizeDb() {
+//		return rxCellSizeDb;
+//	}
+//
+//	public void setRxCellSizeDb(AutoOrManualValue rxCellSizeDb) {
+//		this.rxCellSizeDb = rxCellSizeDb;
+//	}
+//
+//	public AutoOrManualValue getProbeResponseThresholdDb() {
+//		return probeResponseThresholdDb;
+//	}
+//
+//	public void setProbeResponseThresholdDb(AutoOrManualValue probeResponseThresholdDb) {
+//		this.probeResponseThresholdDb = probeResponseThresholdDb;
+//	}
+//
+//	public AutoOrManualValue getClientDisconnectThresholdDb() {
+//		return clientDisconnectThresholdDb;
+//	}
+//
+//	public void setClientDisconnectThresholdDb(AutoOrManualValue clientDisconnectThresholdDb) {
+//		this.clientDisconnectThresholdDb = clientDisconnectThresholdDb;
+//	}
+//
+//	public AutoOrManualValue getEirpTxPower() {
+//		return eirpTxPower;
+//	}
+//
+//	public void setEirpTxPower(AutoOrManualValue eirpTxPower) {
+//		this.eirpTxPower = eirpTxPower;
+//	}
+//
+//	public Boolean getBestApEnabled() {
+//		return bestApEnabled;
+//	}
+//
+//	public void setBestApEnabled(Boolean bestApEnabled) {
+//		this.bestApEnabled = bestApEnabled;
+//	}
+//
+//	public NeighbouringAPListConfiguration getNeighbouringListApConfig() {
+//		return neighbouringListApConfig;
+//	}
+//
+//	public void setNeighbouringListApConfig(NeighbouringAPListConfiguration neighbouringListApConfig) {
+//		this.neighbouringListApConfig = neighbouringListApConfig;
+//	}
+//
+//	public int getMinAutoCellSize(RadioType radioType) {
+//		if (minAutoCellSize == null) {
+//			return radioType == RadioType.is2dot4GHz ? MIN_BG_RADIO_CELL_SIZE : MIN_AC_RADIO_CELL_SIZE;
+//		}
+//
+//		return minAutoCellSize;
+//	}
+//
+//	public void setMinAutoCellSize(Integer minAutoCellSize) {
+//		this.minAutoCellSize = minAutoCellSize;
+//	}
+//
+//	public Boolean getPerimeterDetectionEnabled() {
+//		return perimeterDetectionEnabled;
+//	}
+//
+//	public void setPerimeterDetectionEnabled(Boolean perimeterDetectionEnabled) {
+//		this.perimeterDetectionEnabled = perimeterDetectionEnabled;
+//	}
+//
+//	public ChannelHopSettings getChannelHopSettings() {
+//		return channelHopSettings;
+//	}
+//
+//	public void setChannelHopSettings(ChannelHopSettings channelHopSettings) {
+//		this.channelHopSettings = channelHopSettings;
+//	}
+//
+//	public RadioBestApSettings getBestApSettings() {
+//		return bestApSettings;
+//	}
+//
+//	public void setBestApSettings(RadioBestApSettings bestApSettings) {
+//		this.bestApSettings = bestApSettings;
+//	}
+
+	@Override
+    public RfElementConfiguration clone() {
+        return (RfElementConfiguration) super.clone();
+    }
+	
+	@Override
+	public int hashCode() {
+		final int prime = 31;
+		int result = 1;
+		result = prime * result + ((activeScanSettings == null) ? 0 : activeScanSettings.hashCode());
```

Comment:
It's simpler code to use Objects.hash() on the objects eg Objects.hash(activeScanSettings, beaconInterval...

Reply:
The hashCode and equals were just autogenerated methods from Eclipse, unless we specifically need to adjust it for functionality, I think it should be okay